apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/ArrayAssertionSamples.kt

```diff
@@ -0,0 +1,256 @@
+//TODO remove file with 1.0.0
+@file:Suppress("DEPRECATION")
+
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class ArrayAssertionSamples {
+
+    @Test
+    fun asListFeature() {
+        expect(arrayOf("A", "B")) asList o toBe listOf("A", "B")
+        //                           | subject is now of type List<String>
+    }
+
+    @Test
+    fun asList() {
+        expect(arrayOf("A", "B"))
+            .asList { // subject within this block is of type List<String>
+                it toBe listOf("A", "B")
+            } // subject here is back to type Array<String>
+
+        fails {
+            // all assertions are evaluated inside an assertion group block; for more details:
+            // https://github.com/robstoll/atrium#define-single-assertions-or-assertion-groups
+            expect(arrayOf("A", "B"))
+                .asList {
+                    it contains "C"  // fails
+                    it contains "D"  // still evaluated even though `contains "C"` already fails
+                    //                  use `.asList.` if you want a fail fast behaviour
```

Comment:
```suggestion
                    //                  use `asList o` if you want a fail fast behaviour
```
same same in other places